### ISSUE TITLE
Adding an option to make the PageView call onPageChanged only when the drag ends.

### ISSUE THREAD

**Issue by Norbert515** [flutter/flutter#23682]
This would be a pretty small change and I could do a PR for it.
```
if (notification.depth == 0 && widget.onPageChanged != null && notification is ScrollUpdateNotification)
```
Depending on a boolean `ScrollUpdateNotification` would be replaced with `ScrollEndNotification`.

Opinions?

**Comment by zoechi** [comment]
Fixed by #27365

Please let us know if you're still seeing this with latest `master`

**Comment by timsneath** [comment]
Closed per https://github.com/flutter/flutter/pull/27365#issuecomment-461981580
